tests/functional/adapter/test_persist_docs.py

```diff
@@ -10,6 +11,21 @@
     BasePersistDocsCommentOnQuotedColumn,
 )
 
+_MATERIALIZED_VIEW_PROPERTIES__SCHEMA_YML = """
```

Comment:
Nice test fixture. That's some real [Bobby Tables](https://xkcd.com/327/) stuff.